spec/kitchen/verifier/busser_spec.rb

```diff
@@ -235,10 +237,12 @@ def self.common_powershell_variable_specs
           cmd.must_match regexify(%{version="the_best"})
         end
 
+        # rubocop:disable Metrics/LineLength
```

Comment:
Yep - I tried that as well... Unfortunately it fails the test due to the additional whitespace :( Not sure which one is 'more' correct, if you know what I mean.